doc/xanadu_theme/layout.html

```diff
@@ -201,50 +198,30 @@
                       </a>
                     </li>
                   {%- endif %}
-                  <li style="float: right;" class="showsource">
-                    <a href="{{ pathto('_sources/' + sourcename, true)|e }}" class="reference-external" rel="nofollow">
-                      {{ _('Show Source') }}
-                    </a>
-                  </li>
+                  <!-- <li style="float: right;" class="showsource"> -->
```

Comment:
Might be better to use `{# ... #}` instead of `<!--/ -->` because in the latter case, the template code is still rendered.

Reply:
Oh I didn't know that! Cool.

Although I might just delete this codeblock altogether